### ISSUE TITLE
Feature suggestion: open Windows 7 classic clock when clicking on the clock

### ISSUE THREAD

**Issue by Jevil7452** [dremin/RetroBar#732]
This will especially be useful for Classic Theme users, where single-clicking the clock does not seem to do anything by default.

**Comment by 111ebobo** [comment]
That's how it does in old windows, you need to double click to have access to clock and calendar

**Comment by SL6464** [comment]
> That's how it does in old windows, you need to double click to have access to clock and calendar

This is not correct, at least on Vista.  I just checked in a VM and a single click brings up the clock/calendar pop out.

Agreed with the suggestion - would love to see the 7-style clock/calendar implemented.  It's extremely frustrating that single clicking the clock on Windows 10 brings up the Action Centre.  Double clicking it brings up the "Date and Time" Control Panel applet, but being able to actually view the calendar quickly is very convenient and as stated, was there in Vista.

**Comment by xoascf** [comment]
Working on it!
![dasSevenClockInRetroBar](https://github.com/dremin/RetroBar/assets/28744392/6232880f-9ca0-4e4b-b4be-59bc814a62d2)

**Comment by dremin** [comment]
This is in the latest release.
